code-style/git/README.md

```diff
@@ -250,6 +250,24 @@ For the purposes of this example explaining rebasing, let's say that the branch
 
 Voila! You're done. You've successfully rebased a branch onto the master branch!
 
+Naming Branches
+-------------------------
+
+The way we name branches is similar to the style of our commit message. We use types (feat, fix, docs, etc.) to group our branch names so that we know what part of our workflow the branch belongs to combined with a brief subject to describe what the branch does.
+
+**Type**
+
+The allowed types for a branch name are the same as the commit types above. Branch types are followed by two hyphens.
```

Comment:
I love the idea of adding an internal link! You can do so by hovering the title "The Art of the Commit Message" and clicking the little link icon that shows up to the left. Then copy that URL from the URL bar and put it into a Markdown link like so: `[commit types above](url-here.com)`

Reply:
Perfect!